src/bost/__main__.py

```diff
@@ -36,23 +38,134 @@
     default=None,
 )
 @click.option(
-    "--update-refs",
-    "-r",
-    help="Automatically update the references in the schema files",
+    "--update-refs/--no-update-refs",
+    "-r/-R",
+    help="Automatically update the references in the schema files - github URLs are replaced with relative paths",
     is_flag=True,
     default=True,
 )
+@click.option(
+    "--set-default-version/--no-set-default-version",
+    "-d/-D",
```

Comment:
wie ist da das mapping? `-d` set... und `-D` not-set... ?
Wenn ja, ist dies eine übliche Konvention in CLI Tools?